core/unix/os.c

```diff
@@ -482,6 +482,11 @@ our_libc_errno_loc(void)
  */
 typedef int *(*errno_loc_t)(void);
 
+#ifdef LINUX
+/* Stores whether clone3 is unsupported on the system we're running on. */
+static bool is_clone3_enosys = false;
```

Comment:
I considered making this a bitmap, but didn't as it's not needed right now. But I'm open to changing.

Reply:
I think most existing cases are DR using a syscall, not the app: e..g, SYS_memfd_create.  That could conceivably be moved to init time instead of every call to os_create_memory_file().  Maybe that could be done here; or a comment could be added.

Looking at other new syscalls the app uses: rseq has only post-syscall handling.

Reply:
I added a comment for now, to not block the release work on cleanup.